app/views/layouts/_navbar.html.erb

```diff
@@ -21,7 +21,7 @@
         </li>
       <% end %>
 
-      <% if user_signed_in? && (current_user.member? || current_user.admin?) %>
+      <% if user_signed_in? && (current_user.member? || current_user.admin? || current_user.moderator?) %>
```

Comment:
This logic is not great, but cleaning it up will work nicely with a rename of the `member` role. Then there can be a `member?` method on the user that checks for admin, agent, or moderator.

I know I got the okay on that idea in the discord, but figured I'd put it with a separate ticket to keep control this PR's scope.

Reply:
awesome -- maybe in the separate ticket, we can also do a migration to make the enums in order of permissions, from highest to lowest (or vice versa)

Reply:
wouldn't this be better to push into the `user_only` policy later on. As for the migration to make the enums in order I will reject that right now. Generally speaking, you don't change the numbers on those enums ever.

Reply:
in that case, the order clause on the `has_one :current_state` on `UserMenteeApplication` should be on `created_at`, not status, right?

Reply:
it can be on either for that one cause we explicitly changed that already.